.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,144 @@
+version: '3.9'
+
+services:
+  acorn.test:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: roots/dev-8.2
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    environment:
+      PHP_IDE_CONFIG: '${PHP_IDE_CONFIG:-serverName=acorn}'
+      XDEBUG_MODE: '${XDEBUG_MODE:-develop,debug}'
+      XDEBUG_CONFIG: '${XDEBUG_CONFIG:-client_port=9003 client_host=host.docker.internal discover_client_host=true}'
+      REPOSITORY_URL: '${REPOSITORY_URL:-https://github.com/roots/bedrock.git}'
+    volumes:
+      - ~/.ssh:/home/vscode/.ssh:ro
```

Comment:
I tried using SSH agent forwarding, but it didn't work for me. I tried running an agent in Windows. I tried while running an agent in WSL. I tried with an agent running in both Windows and WSL. In all scenarios, the agent was confirmed running but I did not have access to GitHub inside the container.

So this line is here as a workaround, with an understanding that this is a rather limited approach.